app/package.json

```diff
@@ -10,14 +10,16 @@
     "@material-ui/icons": "^4.5.1",
     "@material-ui/lab": "^4.0.0-alpha.39",
     "@material-ui/styles": "^4.8.2",
+    "axios": "^0.20.0",
     "bootstrap": "^4.4.1",
     "downshift": "^3.4.8",
     "font-awesome": "^4.7.0",
     "material-table": "^1.57.2",
+    "package.json": "^2.0.1",
     "prop-types": "^15.7.2",
     "ramda": "^0.27.0",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "^16.13.1",
```

Comment:
@tamarameghla ,
Any reason for updating the version?
Anyway, with the ^ mark, it will refer to the latest minor+patch version of the particular package.